ras_party/support/session_decorator.py

```diff
@@ -21,12 +21,16 @@ def handle_session(f, args, kwargs):
         logger.info("Committing database session.")
         session.commit()
         return result
+    except ClientError:
+        logger.info(f"Rolling back database session due to a ClientError")
+        session.rollback()
+        raise
     except RasError:
-        logger.error(f"Rolling back database session due to failure executing function")
+        logger.info(f"Rolling back database session due to failure executing function")
         session.rollback()
         raise
     except Exception:
-        logger.error("Rolling back database session due to uncaught exception")
+        logger.info("Rolling back database session due to uncaught exception")
```

Comment:
We hit these quite a lot at the moment don't we? Would it be better for these to be at `debug` level?